tests/package-custom-coverage-test.js

```diff
@@ -117,55 +112,49 @@ describeApplication('PackageCustomCoverage', () => {
     });
 
     it.always('does not display the custom coverage section', () => {
-      expect(PackageShowPage.customCoverage).to.equal('');
+      expect(PackageShowPage.hasCustomCoverage).to.be.false;
     });
 
     it('should display a button to add custom', () => {
-      expect(PackageShowPage.$customCoverageAddButton).to.exist;
+      expect(PackageShowPage.hasCustomCoverageAddButton).to.be.true;
     });
 
     describe('clicking on the add custom coverage button', () => {
       beforeEach(() => {
-        PackageShowPage.clickCustomCoverageAddButton();
+        return PackageShowPage.clickCustomCoverageAddButton();
       });
 
       it('should remove the add custom coverage button', () => {
-        expect(PackageShowPage.$customCoverageAddButton).to.not.exist;
+        expect(PackageShowPage.hasCustomCoverageAddButton).to.be.false;
       });
 
       it('displays custom coverage date inputs', () => {
-        expect(PackageShowPage.$beginDateField).to.exist;
-        expect(PackageShowPage.$endDateField).to.exist;
+        expect(PackageShowPage.dateFields.hasBeginDateField).to.be.true;
+        expect(PackageShowPage.dateFields.hasEndDateField).to.be.true;
       });
 
       describe('entering valid coverage', () => {
-        beforeEach(() => {
-          return convergeOn(() => {
-            expect(PackageShowPage.$beginDateField).to.exist;
-            expect(PackageShowPage.$endDateField).to.exist;
-          });
-        });
-
         describe('with begin date and end date', () => {
           beforeEach(() => {
-            PackageShowPage.inputBeginDate('12/16/2018');
-            PackageShowPage.inputEndDate('12/24/2018');
+            return PackageShowPage.dateFields.fillBeginDate('12/16/2018')
+              .append(PackageShowPage.dateFields.fillEndDate('12/24/2018'));
```

Comment:
Shouldn't need `.append` here. Chaining from `dateFields` to other `dateFields` methods should work.

```
PackageShowPage.dateFields
  .fillBeginDate('12/16/2018')
  .fillEndDate('12/24/2018');
```